.circleci/config.yml

```diff
@@ -331,6 +370,36 @@ jobs:
               $host.SetShouldExit(1)
             }
 
+  merge-release-branch:
+    parameters:
+      e:
+        type: executor
+    executor: << parameters.e >>
+    steps:
+      - slack/notify:
+          channel: 'pdt_releases'
+          color: '#FFDEAD'
+          message: 'Merging release branch into main'
+      - ssh-config
+      - checkout
+      - gh-config
+      - prepare-release-environment-variables
+      - merge-validation
+      - run:
+          name: 'Rebasing main off of release branch'
+          command: |
+            echo "Rebasing main off of release branch ${RELEASE_VERSION}"
+            git checkout main
+            git rebase -Xtheirs release/v${RELEASE_VERSION}
+            git push -f
```

Comment:
Unsure if we should really have this set as a force push too. I hadn't seen any issues with just `git push` until yesterday evening. The error I was coming across in my forked repo was:
`To github.com:sfsholden/salesforcedx-vscode-sholden.git
 ! [rejected]          main -> main (non-fast-forward)
error: failed to push some refs to 'git@github.com:sfsholden/salesforcedx-vscode-sholden.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart.`

This was the only way I found to get around it.

Reply:
Makes sense to me, i've seen that force push is usually necessary for rebases

Reply:
We should figure out a way of accomplishing this without a force push. We've disabled force pushes for the `develop` and `main` branches because of how destructive it could be. There are some ways of recovering from those type of issues but that relies on having access to the git info from the machine that ran the force push or having a backup somewhere else. Because of this I do not think force pushing from CircleCI is viable.